tests/plugins/test_plugin.py

```diff
@@ -99,7 +99,7 @@ class AirflowTestPlugin(AirflowPlugin):
     flask_blueprints = [bp]
     appbuilder_views = [v_appbuilder_package]
     appbuilder_menu_items = [appbuilder_mitem]
-    stat_name_handler = staticmethod(stat_name_dummy_handler)
+    stat_name_handler = stat_name_dummy_handler
```

Comment:
This change seems unrelated?

Reply:
It could be because mypy was complaining about this otherwise?

Reply:
That's right @ashb, and mypy might be right to complain about `staticmethod` here, since `stat_name_dummy_handler` is a function and not a method, and already has the expected signature.